### ISSUE TITLE
Changing accessibility theme fails when 'htaccess.RewriteBase' is set in config.php

### ISSUE THREAD

**Issue by eneiluj** [nextcloud/server#13832]
### Steps to reproduce
1. Be sure there is `'htaccess.RewriteBase' => '/',` in config.php and the htaccess is up to date
2. Go to settings -> accessibility and enable dark theme

### Expected behaviour
The theme should immediately be retrieved and applied.

### Actual behaviour
The request to `https://mynextcloud.org/apps/accessibility/css/user-style.css?v=1548460605492` leads to a 404 response code even though the response payload is correct and contains css. Received css content is therefore not processed.

If i remove the RewriteBase option from config.php and regenerate the htaccess, it works fine, the request to `https://mynextcloud.org/index.php/apps/accessibility/css/user-style.css?v=1548460605492` in successful and processed. Accessibility theme is applied.

### Server configuration

**Operating system**:
Debian testing

**Web server:**
Apache 2.4.37-1

**Database:**
SQLite

**PHP version:**
php7.3-fpm (7.3.1-1)

**Nextcloud version:** (see Nextcloud admin page)
15.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated from 15.0.1 which came from 15.0.0 etc...

**Where did you install Nextcloud from:**
Zip archive provided on nextcloud.com

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:                                                                                                                                                             [4/9363]
  - accessibility: 1.1.0
  - activity: 2.8.2
  - admin_audit: 1.5.0
  - calendar: 1.6.4
  - cloud_federation_api: 0.1.0
  - comments: 1.5.0
  - contacts: 3.0.2
  - dav: 1.8.1
  - federatedfilesharing: 1.5.0
  - federation: 1.5.0
  - files: 1.10.0
  - files_external: 1.6.0
  - files_pdfviewer: 1.4.0
  - files_sharing: 1.7.0
  - files_texteditor: 2.7.0
  - files_trashbin: 1.5.0
  - files_versions: 1.8.0
  - files_videoplayer: 1.4.0
  - firstrunwizard: 2.4.0
  - gallery: 18.2.0
  - gpxedit: 0.0.10
  - gpxmotion: 0.0.8
  - gpxpod: 3.0.1
  - impersonate: 1.2.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.3.0
  - news: 13.0.3
  - nextcloud_announcements: 1.4.0
  - notes: 2.5.1
  - notifications: 2.3.0
  - oauth2: 1.3.0
  - password_policy: 1.5.0
  - phonetrack: 0.4.2
  - provisioning_api: 1.5.0
  - serverinfo: 1.5.0
  - sharebymail: 1.5.0
  - social: 0.1.3
  - spreed: 5.0.1
  - support: 1.0.0
  - survey_client: 1.3.0
  - systemtags: 1.5.0
  - theming: 1.6.0
  - twofactor_backupcodes: 1.4.1
  - updatenotification: 1.5.0
  - workflowengine: 1.5.0
Disabled:
  - encryption
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
<?php
$CONFIG = array (
  'instanceid' => '',
  'passwordsalt' => '',
  'datadirectory' => '',
  'dbtype' => 'sqlite3',
  'version' => '15.0.2.0',
  'installed' => true,
  'theme' => '',
  'maintenance' => false,
  'trusted_domains' =>.
  array (
    0 => '',
    1 => '',
  ),
  'secret' => '',
  'mail_from_address' => '',
  'mail_smtpmode' => '',
  'mail_domain' => '',
  'loglevel' => 0,
  'trashbin_retention_obligation' => 'auto',
  'htaccess.RewriteBase' => '/',
  'overwrite.cli.url' => '',
  'overwriteprotocol' => 'https',
  'appstoreenabled' => true,
  'logtimezone' => 'Europe/',
  'mail_smtpsecure' => 'tls',
  'mail_smtphost' => '',
  'mail_smtpport' => '',
  'mail_smtpauthtype' => '',
  'mail_smtpauth' => 1,
  'mail_smtpname' => '',
  'mail_smtppassword' => '',
  'simpleSignUpLink.shown' => false,
);
```
</details>

### Client configuration
**Browser:**
Firefox 66 nightly

**Operating system:**
Ubuntu 18.10

#### Web server error log
<details>
<summary>Web server error log</summary>
Nothing when the problem happens.

</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
Nothing when the problem happens.
</details>

#### Browser log
<details>
<summary>Browser log</summary>

b) The network log : 404 response on request to 

https://mynextcloud.org/apps/accessibility/css/user-style.css?v=1548460605492

If i remove `'htaccess.RewriteBase' => '/',` and regenerate the htaccess, it works fine, the request becomes : 

https://mynextcloud.org/index.php/apps/accessibility/css/user-style.css?v=1548460605492
</details>

**Comment by SR4ven** [comment]
Having exactly the same issue.
RewriteBase is active because I have nc installed in a subdir.

**Comment by kesselb** [comment]
Do you still see this? I remember some changes to regenerate the files if dark theme is enabled. Thanks :+1:

**Comment by SR4ven** [comment]
The dark theme now somewhat works, but the problem still exists when enabling it.

Enabling the dark theme still results in the 404 response described previously. The theme doesn't change at this point.
However, reloading the website shows no more 404 and switches the theme as expected.

**Comment by SR4ven** [comment]
Seems to be fixed in NC18
